src/tests/ftest/util/file_count_test_base.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import re
+from ior_test_base import IorTestBase
+from mdtest_test_base import MdtestBase
+from avocado.core.exceptions import TestFail
+
+
+def get_rf(oclass):
```

Comment:
Can we have this method under different utils files where it can be easily accessible and visible if need to be used by anybody else? May be general_utils.py or something?